src/main/scala/cognite/spark/v1/NumericDataPointsRdd.scala

```diff
@@ -95,7 +95,7 @@ case class NumericDataPointsRdd(
       case _ =>
         countStart match {
           case Some(start) =>
-            val end = countEnd.getOrElse(start.plus(granularity.amount, granularity.unit))
+            val end = countEnd.getOrElse(start.plus(granularity.amount, granularity.unit)).plusMillis(1)
```

Comment:
The comment about this being exclusive is helpful, as it's not obvious why we're doing `.plusMillis(1)` otherwise.

Reply:
Good catch!